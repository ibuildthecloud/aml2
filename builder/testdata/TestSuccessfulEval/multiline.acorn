x: """
    asfd	j
    """
y: """asdf\(x)
fff"""
