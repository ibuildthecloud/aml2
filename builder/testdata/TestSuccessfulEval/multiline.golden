`{
  "x": "asfd\tj\n",
  "y": "asdfasfd\tj\n\nfff"
}`
